Client/Frontend/Browser/Tab.swift

```diff
@@ -985,6 +988,38 @@ extension Tab {
   func injectSessionStorageItem(key: String, value: String) {
     self.webView?.evaluateSafeJavaScript(functionName: "sessionStorage.setItem",
                                          args: [key, value],
-                                         contentWorld: .page)
+                                         contentWorld: BraveSkusScriptHandler.scriptSandbox)
+  }
+}
+
+// MARK: Script Injection
+extension Tab {
+  func setScript(script: UserScriptManager.ScriptType, enabled: Bool) {
```

Comment:
Just a suggestion: Maybe we can change this to use a set that way we can avoid unnecessary reloads of scripts in cases when we can inject multiple scripts at the same time.  otherwise you will add one script then below it you might add another script and it reloads the scripts twice whereas if you add 2 as part of a set it will only reload once.